### PR TITLE
Enable some StyleCop documentation rules

### PR DESCRIPTION
**Pull request by bricelam**

### CODE REVIEW

File: src/Microsoft.Data.Sqlite/SqliteCommand.cs [hunk 1]
```diff
@@ -67,6 +67,7 @@ public SqliteCommand(string commandText, SqliteConnection connection, SqliteTran
         /// Gets or sets a value indicating how <see cref="CommandText" /> is interpreted. Only
         /// <see cref="CommandType.Text" /> is supported.
         /// </summary>
+        /// <value>a value indicating how <see cref="CommandText" /> is interpreted.</value>
```
Comment (bricelam) [inline src/Microsoft.Data.Sqlite/SqliteCommand.cs]: Oops, will capitalize.

File: src/Microsoft.Data.Sqlite/Microsoft.Data.Sqlite.ruleset [hunk 2]
```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RuleSet Name="Rules for Microsoft.Data.Sqlite" Description="Code analysis rules for Microsoft.Data.Sqlite." ToolsVersion="14.0">
+  <Rules AnalyzerId="StyleCop.Analyzers" RuleNamespace="StyleCop.Analyzers">
+    <Rule Id="SA1000" Action="None" />
```
Comment (natemcmaster) [inline src/Microsoft.Data.Sqlite/Microsoft.Data.Sqlite.ruleset]: Never used rulesets....is there a way to ignore all except SA1609?

**Comment by natemcmaster** [comment]
LGTM. Do we plan to enable the same thing on other projects?

File: src/Microsoft.Data.Sqlite/Microsoft.Data.Sqlite.ruleset (hunk 2 above)
Comment (bricelam) [inline src/Microsoft.Data.Sqlite/Microsoft.Data.Sqlite.ruleset]: Rules have a default action. This disables most that are on by default and enables that one that's off by default. I'll see if there's a way to be explicit about which ones are enabled.

**Comment by bricelam** [comment]
I've proposed it to the engineering team. We'll see what happens. We'll at least enable some of them on EF (I have a branch in progress)

**Comment by natemcmaster** [comment]
Ok. :shipit: for this project.

File: src/Microsoft.Data.Sqlite/Microsoft.Data.Sqlite.ruleset (hunk 2 above)
Comment (bricelam) [inline src/Microsoft.Data.Sqlite/Microsoft.Data.Sqlite.ruleset]: There is no way to do this. You can make the default action of rules stricter, but you can't relax it.

**Pull request merged**